include/solvers/initial_conditions.h

```diff
@@ -34,7 +34,8 @@ namespace Parameters
     L2projection,
     viscous,
     nodal,
-    ramp
+    ramp,
+    average_velocity_profile
```

Comment:
I would add the documentation for this in the current PR in the rst file. It is generally not a good habit to add features without documenting them from the get go.
Quick question, this average_velocity_profile needs to reside on the same mesh as the simulation am I right? In the future, we could also have it reside on any mesh.

Reply:
Yes, for the moment it needs to reside on the same mesh